.code-samples.meilisearch.yaml

```diff
@@ -423,25 +423,27 @@ documents_guide_add_movie_1: |-
       title: "Amélie Poulain".to_string(),
     }
   ], None).await.unwrap();
-document_guide_create_index_primary_key: |-
-  client.create_index("movies", Some("reference_number")).await.unwrap();
-document_guide_add_document_primary_key: |-
+primary_field_guide_update_document_primary_key: |-
+  client.index("books").update(Some("title")).await.unwrap();
```

Comment:
Why don't use the method `UpdateIndex` ?

Reply:
it doesnt exists haha

Reply:
My bad ... 😄